### PR TITLE
Fb subpage

### PR DESCRIPTION
**Pull request by sourishg**

### CODE REVIEW

**Comment by narayanaditya95** [comment]
First of all, remove commits that are not meant to be included in this PR.
Considering the remote to Robotix/Robotix.github.io is named ```upstream```.
```bash
git reset --hard upstream/master
git cherry-pick ceb4c36f5e4d2cfde94511048ada9b582c6fd1fb
```

File: _includes/facebook-feed.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<div id="fb-root"></div>
+<script>(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.4";
+  fjs.parentNode.insertBefore(js, fjs);
+}(document, 'script', 'facebook-jssdk'));</script>
+
+<section id="twitter-feed" class="twitter-feed image-section parallax">
```
Comment (narayanaditya95) [inline _includes/facebook-feed.html]: How about changing the id and css class names

File: _includes/facebook-feed.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!-- Begin Facebook Feed Section -->
+<div id="fb-root"></div>
+<script>(function(d, s, id) {
```
Comment (narayanaditya95) [inline _includes/facebook-feed.html]: What functionality does this script add?
Does it poll facebook posts with the hashtag #robotix?

**Comment by narayanaditya95** [comment]
@sourishg Can you please push this commit to your gh-pages branch so that I can see it live on sourishg.robotix.in/Robotix.github.io

File: _includes/facebook-feed.html (hunk 2 above)
Comment (sourishg) [inline _includes/facebook-feed.html]: The script is required for integrating any facebook plugins to a website.

File: _includes/facebook-feed.html (hunk 1 above)
Comment (sourishg) [inline _includes/facebook-feed.html]: I don't know the css structure well as of now. I say let it be for now. Will change it later.

File: _includes/facebook-feed.html (hunk 2 above)
Comment (narayanaditya95) [inline _includes/facebook-feed.html]: Its 172kb big. Our entire homepage (with all images, sprites, CSS) is under 400kb.
It is a serious NO from my end.

File: _includes/facebook-feed.html (hunk 1 above)
Comment (narayanaditya95) [inline _includes/facebook-feed.html]: Peace

File: _includes/twitter-feed.html [hunk 3]
```diff
@@ -1,15 +0,0 @@
-<!-- Begin Twitter Feed Section -->
-<section id="twitter-feed" class="twitter-feed image-section parallax">
-    <div class="overlay solid-overlay"></div>
-    <div class="content-wrapper">
-        <div class="col-md-8 col-md-offset-2">
-            <svg class="icon" viewBox="0 0 30 32">
-                <use xlink:href="{{ site.baseurl }}/img/sprites/sprites.svg#icon-twitter"></use>
-            </svg>
-            <h3>{{ site.tweetsFeedTitle }} <a href="https://twitter.com/search?q=%23{{ site.twitterHashTag }}" target="_blank">#{{ site.twitterHashTag }}</a>
```
Comment (narayanaditya95) [inline _includes/twitter-feed.html]: Remove the anchor tag. Searching for robotix hastag on facebook does not reveal TRS.

File: _includes/facebook-feed.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}(document, 'script', 'facebook-jssdk'));</script>
+
+<section id="twitter-feed" class="twitter-feed image-section parallax">
+    <div class="overlay solid-overlay"></div>
+    <div class="content-wrapper">
+        <div class="col-md-8 col-md-offset-2">
+            <svg class="icon" viewBox="0 0 30 32">
+                <use xlink:href="{{ site.baseurl }}/img/sprites/sprites.svg#icon-facebook"></use>
+            </svg>
+            <h3>{{ site.FbFeedTitle }} <a href="https://facebook.com/hashtag/{{ site.FbHashTag }}" target="_blank">#{{ site.FbHashTag }}</a>
```
Comment (narayanaditya95) [inline _includes/facebook-feed.html]: Remove the anchor tag. No need to search for #robotix on FB.

**Comment by narayanaditya95** [comment]
Lets make the PR simpler.
The FB section will contain:
- A FB icon at top :white_check_mark: 
- Are you geared up (or something similar) text with no hyperlinks :negative_squared_cross_mark: 
- #robotixiitkgp with a link to our FB page :negative_squared_cross_mark: 

Peace?

**Comment by narayanaditya95** [comment]
@sourishg Thanks.

**Pull request merged**